src/Spark.Engine/Core/IFhirModel.cs

```diff
@@ -59,5 +60,6 @@ public interface IFhirModel
 
         CompartmentInfo FindCompartmentInfo(ResourceType resourceType);
         CompartmentInfo FindCompartmentInfo(string resourceType);
+        ClassMapping FindClassMapping(string resourceType);
```

Comment:
For this version we need to comment `FindClassMapping()` out and instead add a `// FIXME: Introduce FindClassMapping in next major version` then we can instead add this when we go for a major version later. We don't want to break our interfaces within the current version.